tests/component/task/test_task_read_waveform_di.py

```diff
@@ -61,28 +71,26 @@ def test___digital_single_channel___read_waveform_lines_and_port___returns_valid
 
     waveform = di_single_chan_lines_and_port_task.read_waveform(samples_to_read)
 
-    # Note, the data on the port's waveform is MSB instead of LSB because of bug AB#3178052
-    # When that bug is fixed, these asserts should be updated
-    assert _get_waveform_data(waveform) == [0, 1025, 514, 1539, 260, 1285, 774, 1799, 128, 1153]
+    assert _get_waveform_bitstrings(waveform) == [
+        "00000000000",
+        "00100000001",
+        "01000000010",
+        "01100000011",
+        "10000000100",
+        "10100000101",
+        "11000000110",
+        "11100000111",
+        "00000001000",
+        "00100001001",
+    ]
     assert waveform.sample_count == samples_to_read
     assert waveform.channel_name == di_single_chan_lines_and_port_task.di_channels[0].name
-    assert waveform._get_signal_names() == [
```

Comment:
I assume this removes all references to internal methods like `_get_signal_names`.